NEWS.md

```diff
@@ -1,4 +1,12 @@
 ## 10/14/2020 v1.0.0 - Released
 
- * Initial module release with common functionalities for generating records in MARC format 
- * Has capabilities to read records in json format, parse and apply translation functions
\ No newline at end of file
+ * Initial module release with common functionalities for generating records in MARC format
+ * Has capabilities to read records in json format, parse and apply translation functions
+
+ ## 06/11/2020 v1.0.1 Bug Fixes
```

Comment:
Please move this to the top, the latest releases are maintained at the top usually